kolibri/plugins/learn/assets/src/views/LearnIndex.vue

```diff
@@ -132,6 +132,9 @@
       currentPage() {
         return pageNameToComponentMap[this.pageName] || null;
       },
+      pageName() {
```

Comment:
This computed prop is colliding with `...mapState(['pageName']),` above - personally, I think your computed prop is far more reliable here, as I have seen some funky behaviour with the vuex store page name not being what it ought to be.

Should remove the mapState reference though, and will also need to update the tests that rely on mocking `pageName` in vuex state in order to test functionality.

Reply:
I definitely missed that! Based on the conversations on Slack, it seems like we can just remove that `<ActionBarSearchBox />` component completely from the Learn index, so this should be okay.